src/App.jsx

```diff
@@ -36,36 +40,70 @@ function App() {
     setMoves((currentMoves) => currentMoves + 1);
   };
 
+  let rgb = [...rgbStart];
+  if (moves <= 60) {
+    rgb[0] += 3 * moves;
+    rgb[1] -= 3 * moves;
+    rgb[2] -= moves;
+  } else {
+    rgb = [...rgbEnd];
+  }
+
+  const emojiIndex = Math.floor(moves / size);
+
   return (
     <div className="container text-center mt-3">
       <h1>Lights Out</h1>
-      <div className="d-flex justify-content-center my-3">
-        <select
-          className="form-select"
-          aria-label="Size"
-          onChange={onSizeChange}
-        >
-          <option value="5">5x5</option>
-          <option value="6">6x6</option>
-          <option value="7">7x7</option>
-          <option value="8">8x8</option>
-          <option value="9">9x9</option>
-        </select>
-      </div>
 
-      {winAlert ? (
+      {!!winAlert && (
         <div className="alert alert-success" role="alert">
           You won!
         </div>
-      ) : (
-        <></>
       )}
 
-      <div>Moves: {moves}</div>
-      <TileBoard arrayGrid={arrayGrid} onToggle={onToggle} />
-      <button className="btn btn-primary mt-3" onClick={onRestart}>
-        New game
-      </button>
+      <div className="mt-3">
+        <span
+          className="movesCounter"
+          style={{ color: `rgb(${rgb[0]}, ${rgb[1]}, ${rgb[2]})` }}
+          key={`movesCounter: ${moves}`}
+        >
+          {moves}
+        </span>
+        &nbsp;moves
+        <span className="mx-2 emoji" key={`emoji: ${emojiIndex}`}>
+          {emojiIndex < emojis.length ? emojis[emojiIndex] : "😫"}
+        </span>
+      </div>
+
+      <TileBoard
+        arrayGrid={arrayGrid}
+        onToggle={onToggle}
+        key={`tileboard: ${size}`}
+      />
+
+      <div
+        className="d-flex justify-content-center mt-5 mb-3 options"
+        style={{ animationDelay: `${size ** 2 + 4}00ms` }}
```

Comment:
It's probably slightly easier to maintain this if you're doing the full calculation as part of the JS expression (i.e. get rid of `00` in the string portion of this bit).